tests/unit/test_path_template.py

```diff
@@ -413,6 +424,25 @@ def test_transcode_with_wildcard(
             {"data": {"id": 1, "info": "some info"}, "foo": "bar"},
             ["post", "/v1/no/template", {"id": 1, "info": "some info"}, {"foo": "bar"}],
         ],
+        # Single field body with trailing underscore
+        [
+            [["post", "/v1/no/template", "data"]],
+            None,
+            {"data": {"id": 1, "info": "some info"}, "foo_": "bar"},
+            ["post", "/v1/no/template", {"id": 1, "info": "some info"}, {"foo": "bar"}],
+        ],
+        # Single field body with reserved keyword, using message where field name has trailing underscore
+        [
+            [["post", "/v1/no/template", "breakpoint"]],
+            SomeMessage(breakpoint_=Breakpoint(name="test"), debuggee_id="test")._pb,
```

Comment:
minor: could we use distinct values here, say `"test"` and `"exam"`, or `"alice"` and `"bob"`?

Reply:
Fixed.